frontend/tsconfig.json

```diff
@@ -32,7 +32,8 @@
         "moduleResolution": "node",
         "resolveJsonModule": true,
         "isolatedModules": true,
-        "noEmit": true
+        "noEmit": true,
+        "useUnknownInCatchVariables": false,
```

Comment:
I would like to avoid turning useUnknownInCatchVariables off

Reply:
ok, I removed this change